media/css/firefox/whatsnew/whatsnew-nightly.scss

```diff
@@ -47,8 +47,8 @@ body {
         color: var(--body-text-color);
     }
 
-    a:link,
-    a:visited {
+    a:link:not(.mzp-t-product),
+    a:visited:not(.mzp-t-product) {
```

Comment:
I did not test these too deeply into any impact for existing pages if left in afterwards, but expect no impact on any element currently used, light/dark?